core/os/os.h

```diff
@@ -287,6 +288,15 @@ class OS {
 	virtual bool is_in_low_processor_usage_mode() const;
 	virtual void set_low_processor_usage_mode_sleep_usec(int p_usec);
 	virtual int get_low_processor_usage_mode_sleep_usec() const;
+	virtual void set_update_vital_only(bool p_enabled);
+
+	// Convenience easy switch for turning this off outside tools builds, without littering calling code
+	// with #ifdefs. It will also hopefully be compiled out in release.
+#ifdef TOOLS_ENABLED
+	virtual bool is_update_vital_only() const { return _update_vital_only; }
+#else
+	virtual bool is_update_vital_only() const { return false; }
+#endif
```

Comment:
Actually, this means that if users somehow call `OS.set_update_vital_only(true)` in their project, then the game will also skip non-vital updates when running from an editor build (i.e. pressing F5).

I'm not sure that's desired, but thinking further, I'm not sure any of this actually needs to be exposed to the ClassDB if it's only meant for use in the editor.

Reply:
Have removed the core bindings for these functions. :+1: